site/components/ConversationPage.tsx

```diff
@@ -20,9 +24,34 @@ const ConversationPage: React.FC = () => {
     fetchMessages();
   }, [conversationId]);
 
+  const handleCreateConversation = async (e: React.FormEvent) => {
+    e.preventDefault();
+    const response = await fetch('/create_conversation', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorization': `Bearer ${localStorage.getItem('token')}`
+      },
+      body: JSON.stringify({ title, initialMessage, model, temperature })
+    });
+    const data = await response.json();
+    if (response.ok) {
+      setMessages([...messages, { id: data.initialMessageId, content: initialMessage }, { id: data.completionMessageId, content: 'Generated completion' }]);
+    } else {
+      console.error(data);
+    }
+  };
+
   return (
     <div>
       <h2>Conversation</h2>
+      <form onSubmit={handleCreateConversation}>
+        <input type="text" value={title} onChange={(e) => setTitle(e.target.value)} placeholder="Title" required />
+        <input type="text" value={initialMessage} onChange={(e) => setInitialMessage(e.target.value)} placeholder="Initial Message" required />
+        <input type="text" value={model} onChange={(e) => setModel(e.target.value)} placeholder="Model" required />
+        <input type="number" value={temperature} onChange={(e) => setTemperature(parseFloat(e.target.value))} placeholder="Temperature" required />
+        <button type="submit">Create Conversation</button>
```

Comment:
I'd like temperature to be prefilled to `0.0` and model to be prefilled to `gpt-4o`. No title field also.